src/Test/TestAIMuriel.java

```diff
@@ -168,4 +172,134 @@ private boolean testClearAttackRoute()
     return testPassed;
   }
 
+  /** Two units want to go the same way. One is where the other should end up. Make sure they coordinate. */
+  /** START: [ ] [ ] INF [ ] [ ] INF [ ] [ ] [ ] HQ  */
+  /** GOAL : [ ] [ ] [ ] [ ] [ ] INF [ ] [ ] INF HQ */
+  private boolean testWalkInLine()
+  {
+    // Generate a custom tiny map to keep things simple.
+    TerrainType[] HQCol = {TerrainType.HEADQUARTERS};
+    TerrainType[] GRCol = {TerrainType.GRASS};
```

Comment:
Pretty sure we have 2-character abbreviations already defined...

Reply:
We do, but for some reason they are protected members of `IMapBuilder`, so I just did this instead. We could just make those public unless you see a reason not to; they are `static final` anyway, so I'm not really sure why we wouldn't.

Reply:
I don't see any reason to make them non-public either, lol